Functions.Templates/Templates/DurableFunctionsHttpStart-Python-1.x/metadata.json

```diff
@@ -0,0 +1,20 @@
+﻿{
+  "defaultFunctionName": "DurableFunctionsHttpStart",
+  "description": "$DurableFunctionsHttpStart_description",
+  "name": "Durable Functions HTTP starter (upgrade to bundles V2)",
+  "language": "Python",
+  "category": [
+    "$temp_category_core",
+    "$temp_category_durableFunctions"
+  ],
+  "categoryStyle": "other",
+  "enabledInTryMode": false,
+  "userPrompt": [
+    "authLevel"
+  ],
+  "extensions": [
+    {
+      "id": "Microsoft.Azure.WebJobs.Extensions.DurableTask", "version": "2.4.0"
```

Comment:
Same for all of these?

Reply:
Yeah these have been fixed to point to 1.8.6